contracts/multisig-prover/src/contract.rs

```diff
@@ -498,6 +524,39 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_governance_should_confirm_worker_set_without_verification() {
```

Comment:
do we have a test for non-governance to ensure verification is done in that case?

Reply:
yeah the above two tests are for the non-governance case. `test_confirm_worker_set_unconfirmed` tries to confirm as non-governance without verification and expects a failure. And then `test_confirm_worker_set` executes the full happy path flow and confirms a worker set as non governance once the worker set is verified by the voting verifier.